package.json

```diff
@@ -171,13 +170,13 @@
     "lint-staged": "14.0.1",
     "madge": "6.1.0",
     "msw": "1.3.0",
-    "nx": "16.7.4",
+    "nx": "16.8.1",
     "patch-package": "8.0.0",
     "prettier": "3.0.3",
-    "rollup": "3.20.2",
+    "rollup": "3.29.0",
     "rollup-plugin-copy": "3.5.0",
     "rollup-plugin-delete": "2.0.0",
-    "rollup-plugin-dts": "5.3.1",
+    "rollup-plugin-dts": "6.0.1",
```

Comment:
please bump again to new 6.0.2 as there was a bug

Reply:
Done.